packages/common/src/dto-types.ts

```diff
@@ -134,67 +148,46 @@ export class CreateStudentDto {
   concentration?: string;
 }
 
-export class UpdateStudentDto {
-  @IsOptional()
+export class OnboardStudentDto {
   @IsNotEmpty()
   @IsString()
-  fullName?: string;
-
-  @IsOptional()
-  @IsString()
-  nuid?: string;
-
-  @IsOptional()
-  @IsNotEmpty()
-  @IsEmail()
-  email?: string;
+  fullName: string;
 
-  @IsOptional()
-  @IsNotEmpty()
   @IsString()
-  password?: string;
+  nuid: string;
 
-  @IsOptional()
   @IsInt()
   @Min(1898) // the year NEU was established
-  @Max(3000) // will GraduateNU last beyond year 3000?!
```

Comment:
if this was accidentally removed by a replace all, in the future, don't use replace all – it's dangerous because it can unintentionally change code and cause errors later on